### PR TITLE
Translated Java file names to English

### PR DESCRIPTION
**Pull request by BenEklundCS**
I have not changed file contents for this PR, but will begin working on that next.

I'm assuming linker and linter will break if Java class contents do not match their file names?

### CODE REVIEW

**Review by kelvins — COMMENTED** [review]
Could you please just update the README file with the new links?

**Comment by BenEklundCS** [comment]
Sorry about that! Brand new to Github workflows, but now I know. I added another commit to this PR.

What's the easiest way to run these workflows locally/see what results I'd get if I submitted a PR?

**Comment by kelvins** [comment]
No problem. I haven't tested it yet, but there's an open source project called [act](https://github.com/nektos/act) that allows you to run Github Actions locally.

File: README.md [hunk 1]
```diff
@@ -1295,7 +1295,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
                 </a>
             </td>
             <td> <!-- Java -->
-                <a href="./src/java/TorreDeHanoi.java">
+                <a href="./src/java/TowersOfHanoi.java">
```
Comment (kelvins) [inline README.md:1298]: wdyt about using `TowerOfHanoi`? (singular instead of plural)

Comment (BenEklundCS) [reply README.md:1298]: This seems to be the correct title of the algorithm, I'll update this as well as fixing that last linker error tonight. Thanks for the review!!!

**Review by kelvins — APPROVED** [review]

**Pull request merged**